src/components/FormSubmitMessage.js

```diff
@@ -1,16 +1,17 @@
 import styled from 'styled-components/macro';
 
 export default function FormSubmitMessage() {
-  return <StyledMessage>Got it!... but for the moment your notes will be deleted with page refresh 😟</StyledMessage>;
+  return <StyledMessage>Got it!... But f the moment your notes will be deleted at page refresh 😟</StyledMessage>;
```

Comment:
Since you are simply returning a `p` element, is it really useful to have it as separate component? If you need this more than once, it might be a good idea, but if not, you don't need it (and especially because your `FormSubmitMessage` is not a controlled component, so you cannot decide what, e.g., text to render if you use it, it is not really flexible).